src/outputs/parthenon_hdf5.cpp

```diff
@@ -279,11 +279,11 @@ static void writeXdmfSlabVariableRef(std::ofstream &fid, const std::string &name
       // "3" rows for START, STRIDE, and COUNT for each slab with "5" entries.
       // START: iblock variable(_component)  0   0   0
       // STRIDE: 1               1           1   1   1
-      // COUNT:  1               dims[1]     nx3 nx2 nx1
+      // COUNT:  1               1           nx3 nx2 nx1
       fid << prefix << "    "
           << R"(<DataItem Dimensions="3 5" NumberType="Int" Format="XML">)" << iblock
           << " " << i << " 0 0 0 "
-          << " 1 1 1 1 1 1 " << dims[1] << " " << dims321 << "</DataItem>" << std::endl;
+          << " 1 1 1 1 1 1 1 " << dims321 << "</DataItem>" << std::endl;
       writeXdmfArrayRef(fid, prefix + "    ", hdfFile + ":/", name, dims, ndims, "Float",
```

Comment:
My xdmf expertise is still lacking but just one question -- does this per-component chunking correctly handle named variables? In particular, should the argument to this function call be `names[i]` rather than `name`?

Reply:
I'm also far from being an xdmf export -- more like a fire fighter putting out loose ends :D

So the output for AthenaPK now looks like
```
      <Attribute Name="Density" Center="Cell">
        <DataItem ItemType="HyperSlab" Dimensions="64 64 64 ">
          <DataItem Dimensions="3 5" NumberType="Int" Format="XML">0 0 0 0 0  1 1 1 1 1 1 1  64 64 64</DataItem>
          <DataItem Format="HDF" Dimensions=" 45 6 64 64 64" Name="cons" NumberType="Float" Precision="8">
            parthenon.restart.00000.rhdf:/cons</DataItem>
        </DataItem>
      </Attribute>
```
The main `Attribute` name is carrying the component label. I imagine that the `DataItem` with `Format=HDF` here refers to the entry  in the hdf file with with `cons` and that the name itself (of that `DataItem` line) is not relevant.

The current setup works in Visit and Paraview as is,so  I'm in favor of keeping it as is for now and revisit if this is causing problems.

Reply:
OK that makes sense to me (`Attribute` versus `DataItem Name`), thanks for clarifying. Merging now 🤝